app/models/EpicTests.scala

```diff
@@ -23,6 +23,15 @@ object TickerCountType extends Enum[TickerCountType] with CirceEnum[TickerCountT
   case object people extends TickerCountType
 }
 
+sealed trait ContributionFrequency extends EnumEntry
```

Comment:
btw circe can now do enums with slightly neater syntax, see https://github.com/guardian/support-admin-console/blob/main/app/models/SupportFrontendSwitches.scala#L8
I realise lots of places still do it this way though!
E.g.
```
sealed trait ContributionFrequency
case object ONE_OFF extends ContributionFrequency
case object MONTHLY extends ContributionFrequency
case object ANNUAL extends ContributionFrequency
```
and then further down next to the Encoders/Decoders:
```
implicit val stateDecoder: Decoder[ContributionFrequency] = deriveEnumerationDecoder[ContributionFrequency]
implicit val stateEncoder: Encoder[ContributionFrequency] = deriveEnumerationEncoder[ContributionFrequency]
```

Reply:
@tomrf1 thanks, this is v good to know for the future, but I realised that this type had already been created elsewhere in the codebase and named `ContributionType`.

Reply:
oh, good spot!